File: PYTHON/Questões Uri/1094.py
qtd=int(input())
coelho=0
frog=0
mouse=0
casos=0
for cont in range(0,qtd,1):
  quantia,tipo=input().split(" ")
  quantia=int(quantia)
  casos=casos+quantia
  if tipo=="C":
    coelho=coelho+quantia
  if tipo=="S":
    frog=frog+quantia
  if tipo=="R":
    mouse=mouse+quantia

calculoC=((coelho/casos)*100)
calculoS=((frog/casos)*100)
calculoR=((mouse/casos)*100)
print("Total: {} cobaias".format(casos))
print("Total de coelhos: {}".format(coelho))
print("Total de ratos: {}".format(mouse))
print("Total de sapos: {}".format(frog))
print("Percentual de coelhos: %0.2f"%calculoC,"%")
print("Percentual de ratos: %0.2f"%calculoR,"%")
print("Percentual de sapos: %0.2f"%calculoS,"%")

File: JAVASCRIPT/Questões uri/1094.js
const money=(line)=>line.split(" ")
let qtd=+lines.shift()
let coelho=0
let rato=0
let sapo=0
let casos=0
for(let cont=-0;cont<qtd;cont++){
    let[quantia,tipo]=money(lines.shift())
    quantia=parseInt(quantia)
    casos=casos+quantia
    if (tipo=="C"){
        coelho=coelho+quantia
    }
    if(tipo=="R"){
        rato=rato+quantia
    }
    if(tipo=="S"){
        sapo=sapo+quantia
    }
}
let calR=((rato/casos)*100)
let calC=((coelho/casos)*100)
let calS=((sapo/casos)*100)

console.log(`Total: ${casos} cobaias`)
console.log(`Total de coelhos: ${coelho}`)
console.log(`Total de ratos: ${rato}`)
console.log(`Total de sapos: ${sapo}`)
console.log(`Percentual de coelhos: ${calC.toFixed(2)} %`)
console.log(`Percentual de ratos: ${calR.toFixed(2)} %`)
console.log(`Percentual de sapos: ${calS.toFixed(2)} %`)